lib/msf/ui/console/command_dispatcher/resource.rb

```diff
@@ -51,11 +51,12 @@ def cmd_resource(*args)
             end
 
             args.each do |res|
+              res_expand = ::File.expand_path(res)
               good_res = nil
               if res == '-'
                 good_res = res
-              elsif ::File.exist?(res)
-                good_res = res
+              elsif ::File.exist?(res_expand)
+                good_res = res_expand
               elsif
```

Comment:
This condition-less `elsif` is bugging me, even though it's not your code.

Reply:
I'll change that, one minute...